pocs/utils/messaging.py

```diff
@@ -77,14 +72,14 @@ class PanMessaging(object):
 
     ZeroMQ is used to provide the underlying pub-sub support. ZeroMQ
     supports only a very basic message format: an array of bytes.
-    PanMessaging converts the provided message channel and value into
+    PanMessaging converts the provided message topic and value into
     a byte array of this format:
-        <channel-name><space><serialized-value>
+        <topic-name><space><serialized-value>
     """
     logger = get_root_logger()
 
-    # Channel names must consist of the characters.
-    name_re = re.compile('[a-zA-Z][-a-zA-Z0-9_.:]*')
+    # Topic names must consist of the characters.
+    topic_name_re = re.compile('[a-zA-Z][-a-zA-Z0-9_.:]*')
```

Comment:
Don't need the `r'` with `re.compile`?

Reply:
Raw strings are only needed when the string includes characters to be escaped AND you don't want to double escape them... or multi-line strings.